java/server/src/org/openqa/selenium/grid/BUILD.bazel

```diff
@@ -73,6 +73,7 @@ java_export(
         ":base-command",
         "//java/server/src/org/openqa/selenium/cli",
         "//java/server/src/org/openqa/selenium/grid/config",
+        "//java/server/src/org/openqa/selenium/grid/sessionmap/jdbc",
```

Comment:
Is this deliberate? Or a debugging aid?

Reply:
After speaking on Slack, it seems that this is here to help bootstrap things so that new flags appear on the classpath.